onnx2tf/ops/GridSample.py

```diff
@@ -449,7 +449,7 @@ def make_node(
         make_tf_node_info(
             node_info={
                 'tf_op_type': 'GridSample',
-                'tf_inputs': {
+                'tf_inputs': {                                  
```

Comment:
Clear the blanks at the end.

Reply:
Oh, my mistake! I just made a correction.